src/utils.js

```diff
@@ -26,4 +26,20 @@ const sortTable = (data, sortDirection, sortedColumn) => {
   });
 };
 
-export default { getSortColumnAndDirection, filterTable, sortTable };
+const isValidJson = txt => {
+  if (!txt) {
+    return true;
+  }
+  // change single quotes to double quotes
+  const str = txt.replace(/'/g, '"');
```

Comment:
It seems like a nice idea but you can end up with an invalid ex: `{"a": "a'b'c"}` would become invalid. So remove it it will make things simpler.